bugshaker/src/main/java/com/github/stkent/bugshaker/MainActivity.java

```diff
@@ -0,0 +1,279 @@
+package com.github.stkent.bugshaker;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+
+import android.app.Activity;
+import android.app.Application;
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.widget.ImageButton;
+import android.widget.LinearLayout;
+import android.widget.Toast;
+
+import com.github.stkent.bugshaker.flow.email.EmailCapabilitiesProvider;
+import com.github.stkent.bugshaker.flow.email.FeedbackEmailFlowManager;
+import com.github.stkent.bugshaker.flow.email.FeedbackEmailIntentProvider;
+import com.github.stkent.bugshaker.flow.email.GenericEmailIntentProvider;
+import com.github.stkent.bugshaker.flow.widget.DrawingView;
+import com.github.stkent.bugshaker.utilities.LogcatUtil;
+import com.github.stkent.bugshaker.utilities.Logger;
+import com.github.stkent.bugshaker.utilities.Toaster;
+
+public class MainActivity extends AppCompatActivity implements OnClickListener {
+
+	private DrawingView drawView;
+
+	private FeedbackEmailFlowManager feedbackEmailFlowManager;
+
+
+
+	private Activity activity;
+
+	private float smallBrush, mediumBrush, largeBrush, smallEraser, mediumEraser, largeEraser;
```

Comment:
We're using all these only at one other place. These can be used directly there. We don't need to make them class variables.